tests/fixtures/kamereon/vehicles/zoe2_withnav.json

```diff
@@ -0,0 +1,161 @@
+          "deliveryDate": "2020-01-22",
+          "engineEnergyType": "ELEC",
+          "registrationDate": "2020-01-13",
+          "gearbox": {
+            "group": "427",
+            "code": "BVEL",
+            "label": "BOITE A VARIATEUR ELECTRIQUE"
+          },
+          "model": {
+            "group": "971",
+            "code": "X102VE",
+            "label": "ZOE"
+          },
+          "electrical": true,
+          "energy": {
+            "group": "019",
+            "code": "ELEC",
+            "label": "ELECTRIQUE"
+          },
+          "navigationAssistanceLevel": {
+            "group": "408",
+            "code": "SAN408",
+            "label": "CRITERE DE CONTEXTE"
+          },
+          "yearsOfMaintenance": 12,
+          "rlinkStore": false,
+          "radioCode": "",
+          "registrationNumber": "REG-NUMBER",
+          "modelSCR": "ZOE",
+          "easyConnectStore": false,
+          "engineRatio": "605",
+          "battery": {
+            "group": "968",
+            "code": "BT4AR1",
+            "label": "BATTERIE BT4AR1"
+          },
+          "vin": "VF1AAAAA555777999",
+          "retrievedFromDhs": false,
+          "vcd": "ASCOD0/DLIGM2/SSTINC/KITPOU/SKTPGR/SSCCPC/SDPSEC/FDIU2/SSMAP/SSCALL/FACBA1/DANGMO/SSRCAR/SSCABD/AIVCT/AVGSI/ITPK4/VOLNCH/REACTI/AVOSP1/SWALBO/SSDWGE/1234Y/SSAEBS/PRAHL/RRCAM/STANDA/X10/B10/EA3/MD/ELEC/DG/TEMP/TR4X2/AFURGE/RV/ABS/CAREG/LAC/VSTLAR/CPETIR/RET03/PROJAB/RALU16/CEAVRH/ADAC/AIRBA2/SERIE/DRA/DRAP13/HARM02/3ATRPH/SGAV01/BARRAB/TELNJ/SFBANA/KM/DPRPN/AVREPL/SSDECA/ABLAV/ASRESP/ALEVA/SCACBA/SOP02C/STHPLG/SKTGRV/VLCUIR/RETRCR/TRSEV1/RETC/LVAVIP/LVAREL/SASURV/KTGREP/SGSCHA/FRA01/APL03/FSTPO/ALOUC5/CMAR3P/SAN408/NA418/BVEL/AUTAUG/SPREST/RDIF01/ISOFIX/EQPEUR/HRGM01/SDPCLV/CHASTD/TL01A/SPRODI/SAN613/AIRBDE/PSMREC/ELC1/SSPTLP/SANCML/SEXTIN/PE2019/PHAS2/SAN913/THABT2/SSTYAD/SSHYB/052KWH/BT4AR1/VEC018/X102VE/NBT022/5AQ",
+          "firstRegistrationDate": "2020-01-13",
+          "deliveryCountry": {
+            "code": "FR",
+            "label": "FRANCE"
+          },
+          "connectivityTechnology": "RLINK1",
+          "family": {
+            "group": "007",
+            "code": "X10",
+            "label": "FAMILLE X10"
+          },
+          "version": {
+            "code": "INT A MD 1L"
+          }
+        },
+        "status": "ACTIVE",
+        "createdDate": "2020-08-21T16:48:00.243967Z",
+        "cancellationReason": {},
+        "linkType": "OWNER",
+        "connectedDriver": {
+          "role": "MAIN_DRIVER",
+          "lastModifiedDate": "2020-08-22T09:41:53.477398Z",
+          "createdDate": "2020-08-22T09:41:53.477398Z"
+        },
+        "vin": "VF1AAAAA555777999",
+        "lastModifiedDate": "2020-11-29T22:01:21.162572Z",
+        "brand": "RENAULT",
+        "startDate": "2020-08-21",
+        "ownershipStartDate": "2020-01-13",
+        "ownershipEndDate": "2020-08-21"
+      }
+    ],
+    "accountId": "account-id-1"
+  }
```

Comment:
EOF, and please rename "zoe2_withnav.json" > "zoe_50.1.json"

Reply:
All files renamed and fixed !